quarkus/settings.gradle.kts

```diff
@@ -0,0 +1,12 @@
+rootProject.name = "community-ninja-board"
```

Comment:
I don't think we need a context root anymore. This app will have a unique domain/route, so we should clean up the URL by serving from ROOT.

Reply:
This is a gradle construct equivalent to the maven: `<artifactId>community-ninja-board</artifactId>`

Reply:
ok great.

Reply:
@sabre1041 , it's going to be hard to argue for Maven vs Gradle. I am willing to maintain both a Gradle and Maven until we can have enough data to weight the benefits of 1 vs the other. Sounds good?